index.html

```diff
@@ -42,6 +42,176 @@ <h1 class="title">I’m Mariane Glad to see you!</h1>
+                            <span class="dot"></span>
+                            <span class="role">Back End Dev</span>
+                            <span class="dot"></span>
+                            <span class="year">2015</span>
+
+                        </div>
+                        <p class="workDescription">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                        <!-- categories -->
+                        <ul class="worksCategories">
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javaScript</li>
+                        </ul>
+                        <button class="worksBtn">See Project</button>
+                    </div>
+                    <!-- end body -->
+                </div>
+                <!-- end third template -->
+
+                <!-- fourth template -->
+                <div class="worksContainer">
+                    <img src="images/works/work4.png" alt="work4" class="worksImg">
+                    <!-- body -->
+                    <div>
+                        <h1 class="worksBodytitle">Multi-Post
+                            Stories</h1>
+                        <!-- description -->
+                        <div class="frame2">
+                            <span class="name">CANOPY</span>
+                            <span class="dot"></span>
+                            <span class="role">Back End Dev</span>
+                            <span class="dot"></span>
+                            <span class="year">2015</span>
+
+                        </div>
+                        <p class="workDescription">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                        <!-- categories -->
+                        <ul class="worksCategories">
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javaScript</li>
+                        </ul>
+                        <button class="worksBtn">See Project</button>
+                    </div>
+                    <!-- end body -->
+                </div>
+                <!-- end fourth template -->
+
+            </div>
+            <!-- end grid layout container -->
+        </section>
+        <!-- end works section -->
+
+        <!-- about me -->
+        <section id="aboutMe">
+            <h1 class="title">About Myself</h1>
+            <p class="about">I’m a software developer! I can help you build a product , feature or website Look
+                through
+                some
+                of my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <div class="social-title">LET’S CONNECT</div>
```

Comment:
- [ ] This is a link to the contact form so please 🙏🏽 use the `a` tag for the better practices of HTML&CSS.